platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/DescribableBean.java

```diff
@@ -20,7 +20,7 @@
 
 import org.codice.ddf.platform.services.common.Describable;
 
-public abstract class AbstractDescribable implements Describable {
+public class DescribableBean implements Describable {
```

Comment:
Don't know if it's just me but `Bean` feels weird for something you're expected to extend. Do you think we should name it something like `DescribableBase` or `DescribableImpl`?

We should also add a JavaDoc comment on the class explaining how clients are expected to use this class, i.e., as a base class for a service that provides the `Describable` methods rather than as an object meant to be returned.